docker-compose.yml

```diff
@@ -7,4 +7,6 @@ services:
     volumes:
       - ./opt:/opt
       - ./images:/images
+      - ./.ccache:/root/.cache/ccache
+      - ./.buildroot-ccache:/root/.buildroot-ccache
```

Comment:
Can we please also add both folders to .gitignore? Prevents accidental committing those and keeps them out of git status. Otherwise +1. thanks